content/test/integration/service/snapshots/SnapshotManager.spec.ts

```diff
@@ -46,7 +46,8 @@ describe('Integration - Snapshot Manager', () => {
     // Assert snapshot was created
     const snapshotMetadata = snapshotManager.getSnapshotMetadata(EntityType.SCENE)
     expect(snapshotMetadata).toBeDefined()
-    expect(snapshotMetadata!.lastIncludedDeploymentTimestamp).toEqual(lastDeploymentTimestamp)
+    expect(lastDeploymentTimestamp).toBeTruthy((a) => !(a instanceof Error))
```

Comment:
Why `Error`? Also, let's create a helper (maybe in `E2EAssertions`?) that asserts that the deployment was successful